src/bundler.ts

```diff
@@ -5,6 +5,13 @@ import * as vscode from "vscode";
 
 const asyncExec = promisify(exec);
 
+export async function isGemMissing(): Promise<boolean> {
+  const bundledGems = await execInPath("bundle list");
```

Comment:
It looks like we used the `try catch` pattern in `isGemOutdated` and `bundleCheck` functions. Do we want to align with them here? For example:

```js
  try {
    await execInPath("bundle show ruby-lsp");
    return false;
  } catch (exc) {
    return true;
  }
```